pages/common/img2pdf.md

```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF.
+> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Convert images to PDF:
+
+`img2pdf {{path/to/image1.jpg}} {{path/to/image2.jpg}} --output {{path/to/file.pdf}}`
+
+- Convert all PNGs in the current directory to a single pdf:
+
+`img2pdf {{*.png}} --output {{path/to/file.pdf}}`
+
+- Convert only the first page of multi-page images to pdf:
+
+`img2pdf {{path/to/file.gif}} --first-frame-only --output {{path/to/file.pdf}}`
+
+- Auto orient to A4 page size in landscape mode and a border of 2cm horizontally and 5.1cm vertically:
+
+`img2pdf {{path/to/file.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm:5.1cm}} --output {{path/to/file.pdf}}`
+
+- Shrink only larger images to a 10cm by 15cm rectangle inside a 30x20cm page:
+
+`img2pdf {{path/to/file.tiff}} --pagesize {{30cmx20cm}} --imgsize {{10cmx15cm}} --fit {{shrink}} --output {{path/to/file.pdf}}`
+
+- Convert an image to PDF including specific metadata:
+
+`img2pdf {{path/to/file.png}} --title {{title}} --author {{author}} --creationdate {{1970-01-31}} --keywords {{keywords}} --subject {{subject}} --output {{path/to/file.pdf}}`
```

Comment:
Since `--keywords` accepts multiple arguments, I would have to guess the separator. You should indicate it by including multiple keywords. (`keyword1,keyword2` if the separator is `,`)

Reply:
Good point. What about this?
```suggestion
`img2pdf {{path/to/file.png}} --title {{title}} --author {{author}} --creationdate {{1970-01-31}} --keywords {{keyword1,keyword2,...}} --subject {{subject}} --output {{path/to/file.pdf}}`
```

Reply:
you can separate them by a space `--keywords keyword1 keyword2`. These keywords don't show up with Dolphin File Manager, only with `exiftools` and Okular (PDF viewer). Maybe a bug? I don't know.

Reply:
I think we should keep this example rather simple. Also the flag name kind of indicates that it is at least possible to give multiple argument (it's `keywords` and not `keyword`). For more information we still have man pages. 

But if it makes sense sense to write `--keywords {{keyword1 keyword2 ...}}` I see no problem with it.